app/models/live_metric.rb

```diff
@@ -0,0 +1,112 @@
+class LiveMetric < ActsAsArModel
+  set_columns_hash(:timestamp => :datetime)
+
+  class LiveMetricError < RuntimeError; end
```

Comment:
I suggest considering adding the writing to log of all the exceptions, important for troubleshooting